ccd-definition/ComplexTypes/HearingBooking.json

```diff
@@ -0,0 +1,98 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "HearingBooking",
+    "ListElementCode": "hearingType",
```

Comment:
do all of these ListElementCodes need hearing appended to the front. 

hearingBooking.hearingType or hearingBooking.type 🤔 

venue etc make a lot more sense to be just venue etc

Reply:
I think this is an unnesseccary change at this stage and i think having hearing before the field makes it specific to this event so I dont think it does any harm being there